src/index.tsx

```diff
@@ -10,6 +10,7 @@ export {default as FadeIn} from './fade-in';
 
 export {AccordionItem, BoxedAccordionItem, Accordion, BoxedAccordion} from './accordion';
 export {ButtonPrimary, ButtonSecondary, ButtonDanger, ButtonLink, ButtonLinkDanger} from './button';
+export {default as ButtonGroup} from './button-group';
```

Comment:
This is necessary because we are deprecating button/secondaryButton props in `Header` component (can be replaced by `extra` prop from `HeaderLayout).` The issue is that the only way to replicate the usage of button/secondaryButton with extra prop is by using ButtonGroup component. Unless we export it in Mistica, that's not possible (and there are at least 5 scenarios like this in webapp).

I've discussed with design and they believe it's okay to expose this component, because they already have it in the design docs (and it's even exposed in the Mistica storybook nowadays).

@pladaria  @atabel I re-requested your review in order to validate this.